### ISSUE TITLE
Bug with Master Sword!

### ISSUE THREAD

**Issue by higorhgoncalves** [snesrev/zelda3#157]
I've reached the Swamp Palace's Boss and when I pull his minions to attack them with the Sword, it doesn't work, and I noticed that when I am with full life, master sword's beam is not working either, and when I tried to get de Ether Medallion he tells me that I need the Master Sword.

I started the game with the Vita Port made by Rinnegatamante, but tested the save file with the master build in Linux.

### What is your build target?

Linux

**Comment by higorhgoncalves** [comment]
Here it is! [save0.sav](https://drive.google.com/file/d/1kkVWJQEKaBars93s2xZt9RCLrWzsp9le/view?usp=sharing)

**Comment by FitzRoyX** [comment]
the replay desyncs almost immediately, so you can't even watch it to see if he glitched past certain events.

**Comment by higorhgoncalves** [comment]
Well, I played a little bit with the modified snapshot and then I thought, is the Master Sword still in The Woods?
So I went there to check it out, and it really was there. When I got the Master Sword again my goals from the dark world disappeared from the map.
Then I tried to do with my old snapshot and the same happened.
Then I tried to start from the beginning and the bugs did not occurred. But I remembered 2 things that could be the problem, first when I get the master sword I leaved the woods and save and quit, second I didn't go to the sanctuary and went straight to the castle and didn't realize that was without the master sword. But I realized that the game was a little hard.
